app/views/brands/_create_form.html.erb

```diff
@@ -0,0 +1,11 @@
+<%= form_with url: brands_create_path, method: :post do |form| %>
+
+  <div class="field form-group col mb-4">
+    <%= form.label :name, "Brand Name", class: "form-label" %>
+    <%= form.text_field :name, placeholder: "Apple", class: "form-control" %>
```

Comment:
```suggestion
    <%= form.text_field :name, placeholder: "Apple", class: "form-control", required: true %>
```
![image](https://user-images.githubusercontent.com/48946963/125804539-c87610f3-bad3-4593-9cdc-555e60789b55.png)

Reply:
Oh nice, I didn't know about it